{"uid":"4acd9e147ae5ee2d","name":"test_login_logout[data1-edge]","fullName":"tests.test_login#test_login_logout","historyId":"01268ce4d69da7cf97d819af0a9c7b18","time":{"start":1761124530285,"stop":1761124548420,"duration":18135},"status":"failed","statusMessage":"AssertionError: Login failed for user TestUser\nassert False","statusTrace":"browser_name = 'edge', data = {'password': 'Test@123', 'username': 'TestUser'}\n\n    @pytest.mark.parametrize(\"browser_name\", [\"chrome\", \"edge\"])\n    @pytest.mark.parametrize(\"data\", login_data)\n    def test_login_logout(browser_name, data):\n        # Get driver for the current browser\n        driver = DriverFactory.get_driver(browser_name=browser_name, headless=False)\n    \n        login_page = DemoBlazeLogin(driver)\n        login_page.open_url(\"https://www.demoblaze.com/\")\n    \n        success = login_page.login(data['username'], data['password'])\n>       assert success, f\"Login failed for user {data['username']}\"\nE       AssertionError: Login failed for user TestUser\nE       assert False\n\ntests\\test_login.py:53: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Login failed for user TestUser\nassert False","statusTrace":"browser_name = 'edge', data = {'password': 'Test@123', 'username': 'TestUser'}\n\n    @pytest.mark.parametrize(\"browser_name\", [\"chrome\", \"edge\"])\n    @pytest.mark.parametrize(\"data\", login_data)\n    def test_login_logout(browser_name, data):\n        # Get driver for the current browser\n        driver = DriverFactory.get_driver(browser_name=browser_name, headless=False)\n    \n        login_page = DemoBlazeLogin(driver)\n        login_page.open_url(\"https://www.demoblaze.com/\")\n    \n        success = login_page.login(data['username'], data['password'])\n>       assert success, f\"Login failed for user {data['username']}\"\nE       AssertionError: Login failed for user TestUser\nE       assert False\n\ntests\\test_login.py:53: AssertionError","steps":[],"attachments":[{"uid":"ecb2016887cb8e52","name":"stdout","source":"ecb2016887cb8e52.txt","type":"text/plain","size":52}],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":1,"stepsCount":0,"attachmentStep":false},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"host","value":"ASCENDI-1GUJSNA"},{"name":"thread","value":"2564-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser_name","value":"'edge'"},{"name":"data","value":"{'username': 'TestUser', 'password': 'Test@123'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"4acd9e147ae5ee2d.json","parameterValues":["'edge'","{'username': 'TestUser', 'password': 'Test@123'}"]}